SPECS/gpumon.spec

```diff
@@ -1,6 +1,6 @@
 Name:           gpumon
 Version:        0.18.0
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
The first digit from the release belonging to the CH package, we should change the second digit instead, so `1.1`